androidsdk/src/test/java/com/wootric/androidsdk/WootricTest.java

```diff
@@ -115,6 +115,20 @@ public void setUp() {
         assertThat(wootric.endUser.getEmailOrUnknown()).isEqualTo("nps@example.com");
     }
 
+    @Test public void setEndUserExternalId() throws Exception {
```

Comment:
@marinhero if you send "" it will send that empty string on the request, if it's `null` it won't add the parameter to the request.